pages/[category]/index.tsx

```diff
@@ -71,6 +71,7 @@ const LandingPage = ({ preFetchedOrgs }: LandingPageProps) => {
             <MobileButtonsLandingPage
               activeCopy={activeCopy}
               setOpen={setOpen}
+              showFilter={'true'}
```

Comment:
on the whole it looks it is working well. i'm curious about this prop though. is it meant to be a string instead of a boolean. if it is a string do we need it as a prop as a it will always return the same result if it is static.

Reply:
Could be a boolean i suppose it was giving me grief initially so just made it into a string but essentially a boolean to check if we should be displaying it

Reply:
Either way though, if it's a static value it will only render one way, so we don't need it as a prop. unless i'm missing something?

Reply:
Not sure if I am understanding correctly. The filter button should be optional. I am using the showfilter prop as a boolean to see if we should be displaying it. I don't want it to showup in `pages/search/index` but I do want it to show up in `pages/[category]/index`